libbpf-rs/src/map.rs

```diff
@@ -271,6 +273,11 @@ impl Map {
         self.max_entries
     }
 
+    /// Returns the map's id.
+    pub fn id(&self) -> u32 {
```

Comment:
Let's tag it `#[inline]` given the brevity of the method and to make it eligible for inlining across crates.